exporter/sumologicexporter/exporter.go

```diff
@@ -16,44 +16,112 @@ package sumologicexporter
 
 import (
 	"context"
+	"net/http"
 
 	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/component/componenterror"
 	"go.opentelemetry.io/collector/consumer/pdata"
-	"go.uber.org/zap"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
```

Comment:
Just a comment: I've always tried to stick with the following format

```
import (
	"stdlib1"
	"stdlib2"

	"3rdpartylib1"
	"3rdpartylib2"

	"same_repo_package1"
	"same_repo_package2"
)
```

Reply:
vscode should take care about it and it does, but not always